main.py

```diff
@@ -94,10 +102,36 @@
                 hf.create_dataset('y_test', data=y_test)
                 hf.create_dataset('patchSize', data=patchSize)
                 hf.create_dataset('patchOverlap', data=cfg['patchOverlap'])
+                hf.create_dataset('OrigPatchSize', data=OrigPatchSize)
 
     # perform training
     for iFold in range(0, len(X_train)):
         cnn_main.fRunCNN({'X_train': X_train[iFold], 'y_train': y_train[iFold], 'X_test': X_test[iFold], 'y_test': y_test[iFold], 'patchSize': patchSize}, cfg['network'], lTrain, cfg['sOpti'], sOutPath, cfg['batchSize'], cfg['lr'], cfg['epochs'])
+=======
```

Comment:
you are training and afterwards retrain with the scaled patches?
If so: We should put in a separate "training method", see comment above for multiscale training.